Dockerfile

```
@@ -0,0 +1,8 @@
+FROM rust:latest
+
+ENV TZ=Europe/Oslo
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+COPY target/release/fdk-mqa-property-checker /usr/local/bin
+
+CMD /usr/local/bin/fdk-mqa-property-checker --brokers "$BROKERS" --schema-registry "$SCHEMA_REGISTRY"
```

Som vi snakket om kan det være mulig å flytte binary over i et minimalt image. Jeg har ikke testet om det funker.

Krever også at første linje har `FROM rust:latest as build`
```suggestion
FROM gcr.io/distroless/base-debian11
COPY --from=build target/release/fdk-mqa-property-checker /fdk-mqa-property-checker

CMD /fdk-mqa-property-checker --brokers "$BROKERS" --schema-registry "$SCHEMA_REGISTRY"
```